PhotoMap/Features/MapPhoto/Coordinators/MapPhotoCoordinator.swift

```diff
@@ -24,7 +26,7 @@ class MapPhotoCoordinator: Coordinator {
     }
 
     func start(with photo: PhotoDVO) -> UINavigationController {
-        let viewModel = MapPhotoViewModel(coordinator: self, diContainer: DIContainer(), photo: photo)
+        let viewModel = MapPhotoViewModel(coordinator: self, diContainer: diContainer, photo: photo)
```

Comment:
Good find of DIContainer().
Please check the rest of the code as well. We should only use single DIContainer for the entire app.